src/overlays/actors/ovl_Bg_Spot00_Hanebasi/z_bg_spot00_hanebasi.c

```diff
@@ -1,21 +1,36 @@
+/*
+ * File: z_bg_spot00_hanebasi.c
+ * Overlay: ovl_Bg_Spot00_Hanebasi
+ * Description: Hyrule Field Drawbridge and Torches
+ */
+
 #include "z_bg_spot00_hanebasi.h"
 
 #define FLAGS 0x00000010
 
 #define THIS ((BgSpot00Hanebasi*)thisx)
 
+typedef enum {
+    DT_CHAIN_1,
+    DT_CHAIN_2,
+    DT_DRAWBRIDGE = -1,
```

Comment:
It's pretty minor but I suggest laying out the enum like this (with no trailing comma since it triggers a compiler warning)
```suggestion
    /* -1 */ DT_DRAWBRIDGE = -1,
    /*  0 */ DT_CHAIN_1,
    /*  1 */ DT_CHAIN_2
```